buildSrc/src/main/groovy/airbyte-java-connector.gradle

```diff
@@ -18,67 +17,50 @@ class AirbyteJavaConnectorExtension {
         this.project = project
     }
 
+    void setUseLocalCdk(boolean useLocalCdk) {
+        this.useLocalCdk = useLocalCdk
+        addCdkDependencies()
+    }
+
+    static final List<String> IMPLEMENTATION = [
+            'airbyte-commons',
+            'airbyte-json-validation',
+            'airbyte-commons-cli',
+            'airbyte-api',
+            'config-models-oss',
+            'init-oss',
+    ]
+
+    static final List<String> TEST_IMPLEMENTATION = [
+            'airbyte-commons',
+            'airbyte-json-validation',
+            'airbyte-api',
+            'config-models-oss',
+    ]
+
+    static final List<String> INTEGRATION_TEST_IMPLEMENTATION = [
+            'config-models-oss',
+            'init-oss',
+            'acceptance-test-harness',
+    ]
+
     void addCdkDependencies() {
```

Comment:
what happens to all the connectors whose build.gradle calls this?

Reply:
It's a no-op.

Reply:
oh yeah, sorry. I thout you had removed the addCdkDependencies. It just moved below